src/constructs/autoscaling/asg.ts

```diff
@@ -140,6 +141,7 @@ export class GuAutoScalingGroup extends GuAppAwareConstruct(AutoScalingGroup) {
       maxCapacity: maximumInstances ?? minimumInstances * 2,
       minCapacity: minimumInstances,
       groupMetrics: groupMetrics,
+      signals: updatePolicy ? Signals.waitForAll() : undefined,
```

Comment:
IIUC the default signal is 100% success, within 5 minutes. Might we want to allow customisation of this? Or set it relative to the healthcheck?

Reply:
Ah I think I made a mistake here, it should be:

```suggestion
      signals: updatePolicy ? Signals.waitForAll() : signals,
```

So that you can pass a `signals` prop to the ASG to override the default. Having it be based on the healthcheck sounds reasonable, but perhaps we can leave it as the default till we understand it a bit more?